api/admin/controller/patron_auth_services.py

```diff
@@ -16,6 +16,9 @@
 from api.saml.provider import SAMLWebSSOAuthenticationProvider
 from api.simple_authentication import SimpleAuthenticationProvider
 from api.sip import SIP2AuthenticationProvider
+from api.sirsidynix_authentication_provider import (
+    SirsiDynixHorizonAuthenticationProvider,
```

Comment:
We will be using this integration with both Horizon and Symphony, so it probably makes sense to just call this `SirsiDynixAuthenticationProvider` unless we end up needing two different implementations, one for Horizon and one for Symphony.